src/openapi/streaming/streaming.js

```diff
@@ -933,6 +934,36 @@ Streaming.prototype.disconnect = function() {
     this.connection.stop();
 };
 
+Streaming.prototype.pause = function() {
+    this.paused = true;
+
+    if (this.reconnecting) {
+        clearTimeout(this.reconnectTimer);
+        this.reconnecting = false;
+        this.retryCount = 0;
+    }
+
+    this.orphanFinder.stop();
+
+    for (let i = 0; i < this.subscriptions.length; i++) {
+        const subscription = this.subscriptions[i];
+        // Mark the subscription as not having a connection and reset it so its state becomes unsubscribed
+        subscription.onConnectionUnavailable();

```

Comment:
We do not need to unsubscribe everything?

Reply:
reset takes care of that
it checks subscriptions status before subscribing / unsubscribing

Reply:
yes but it will unsubscribe when you come to resume.
However, closing the connection should clean up 🤞 

I wonder if we swap these round - the unsubscribe will be kicked off as it is sync, but further commands like subscribe will not.